project4part2/presentation/current.tex

```diff
@@ -40,8 +40,11 @@ \subsection{Terminology}
 \begin{frame}
 \frametitle{}
 Each (unpadded) \textdef{text} is a string.
-Padding and unpadding were already described in
-the project 4 progress report at: \\ \url
+Padding is a technique to make a string equal in length to
+the key size. If this were not so, the security of block
+ciphers would be compromised.
```

Comment:
my padding/unpadding functions only convert a string to/from an integer, without any bit-count guarantee